src/socket-activate/socket-activate.c

```diff
@@ -452,7 +449,7 @@ static int run(int argc, char **argv) {
         if (r <= 0)
                 return r;
 
-        exec_argv = strv_copy(arg_args);
+        exec_argv = strv_copy(argv + optind);
```

Comment:
strv_skip() ?

Reply:
We use the current form across the codebase, since optind is pre-determined to be within bound.